Formula/w/wireshark.rb

```diff
@@ -94,6 +85,19 @@ def caveats
   end
 
   test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <stdio.h>
+      #include <ws_version.h>
+
+      int main() {
+        printf("%d.%d.%d", WIRESHARK_VERSION_MAJOR, WIRESHARK_VERSION_MINOR,
+               WIRESHARK_VERSION_MICRO);
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}/wireshark", "-o", "test"
+    output = shell_output("./test")
+    assert_equal version.to_s, output
```

Comment:
Let's give it some space
```suggestion
    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}/wireshark", "-o", "test"
    assert_equal version.to_s, shell_output("./test")

```